src/main/java/com/example/petbuddybackend/dto/notification/SimplyNotificationDTO.java

```diff
@@ -0,0 +1,25 @@
+package com.example.petbuddybackend.dto.notification;
+
+import com.example.petbuddybackend.entity.notification.ObjectType;
+import com.example.petbuddybackend.entity.user.Role;
+import lombok.*;
+import lombok.experimental.SuperBuilder;
+
+import java.util.Set;
+
+@SuperBuilder
+@Getter
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
+public class SimplyNotificationDTO extends NotificationDTO {
+    @Builder.Default
+    private String dType = "Notification";
```

Comment:
Dwa pytania:
- nie chcemy żeby to był enum?
- może nazwa `type` zamiast `dType`? Ale pytanie wynika bardziej  z tego że nie wiem jakim skrótem jest `d`

Reply:
tak sie czesto okresla typ klasy, JPA tez tak robi w przypadku różnych relacji w dziedziczeniu i pole, które określa objekt klasy podrzędnej to dType, wiec tak też zrobiłem.
W sumie możemy enuma dac

Reply:
a dtype to data type